src/common.tests/Logging/ExtensionsTests.cs

```diff
@@ -23,30 +23,6 @@ public class ExtensionsTests
                 { "Authorization", new List<string>() { "authorization" } },
             };
 
-        [Fact]
-        public void SerializeAggregateException()

```

Comment:
@Eneuman since this was removed from ```Extensions.cs``` and added to JsonHelper, can you add them to JsonHelperTests please ?

Reply:
@hsubramanianaks Fixed